forte/utils/utils.py

```diff
@@ -303,3 +304,45 @@ def build_alignment(self, text: str, text_to_align: str):
 
     def _get_opcodes(self):
         yield from self.__matcher.get_opcodes()
+
+
+def create_import_error_msg(
+    extra_module: str,
+    forte_module: str,
+    component_name: str,
+):
+    """
+    Create an error message for importing package extra required by a forte
+    module.
+
+
+    Args:
+        extra_module: module name should be installed by pip.
+        forte_module: forte module User should install by
+            ``pip install forte[`forte_module`]`` to install all
+            extra packages for using the forte module.
+        component_name: the forte component that needs the module.
+
+    """
+    forte_module_link_d = {
+        "data_aug": "https://github.com/asyml/forte/blame/master/README.md#L70",
```

Comment:
These links might be updated from time to time, like line number changes. Also, you should not use the `blame` version, even if this points to the correct line. 

A couple of solutions would be:
1. Consider adding link/anchor to that line
2. Or simply use a tagged version or a particular commit

You can point to the general area, without pointing to the specific line if that's not possible.

Reply:
To have anchors for these links, there should be section for each forte module like the screenshot below. Do you think it's acceptable? If so I will create a separate PR for it.
![image](https://user-images.githubusercontent.com/28021889/165018556-5e835b6e-e948-43c8-a502-85cfa46a9d34.png)

Reply:
i don't think we have to point to the exact line here, just point to this section for now until we know a better solution.